code/src/index.js

```diff
@@ -1,6 +1,513 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
-import { App } from './App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+
```

Comment:
As with your css, you need to go over your indentation and extra empty lines. Like I commented earlier, I find it a little difficult to follow your code - maybe next time you can practice breaking functionality out in their own components?